logstash-8.yaml

```diff
@@ -28,7 +28,8 @@ package:
     runtime:
       - bash # some helper scripts use bash and busybox utilities
       - busybox
-      - openjdk-17-default-jvm
+      - openjdk-17
+      - openjdk-17-default-jdk
```

Comment:
This is fine, and will work, but `-default-jdk` pulls in `-openjdk-17` anyway so we don't need to include it twice